core/modules/file/file.module

```diff
@@ -1316,6 +1316,9 @@ function file_managed_file_process($element, &$form_state, $form) {
   );
 
   // Add a button that will be used to select existing files if enabled.
+  // Attaching the assets from filter module makes sure, the browser works
+  // even if no field on that page has a filter set.
+  $filter_path = backdrop_get_path('module', 'filter');
```

Comment:
There's a comment on line 1318 that explains this whole block of code, but then this line sits between the two. Maybe this line can be moved above the comment, or even to the top of the function...?

Reply:
Ehm.. so where should I put it? Above? Next to the `#attached` - I'm confused. :wink: